test/TEST-03-JOBS/test-jobs.sh

```diff
@@ -65,10 +65,10 @@ EOF
 
 # wait2 succeeds
 START_SEC=$(date -u '+%s')
-systemctl start --wait wait2.service || exit 1
+systemctl start --wait wait2.service
 END_SEC=$(date -u '+%s')
 ELAPSED=$(($END_SEC-$START_SEC))
-[[ "$ELAPSED" -ge 2 ]] && [[ "$ELAPSED" -le 4 ]] || exit 1
+[[ "$ELAPSED" -ge 2 ]] && [[ "$ELAPSED" -le 4 ]]
```

Comment:
The script won't fail If `$ELAPSED` is less then `2`. I double-checked that by running the following command
```
$ bash -x -e -c 'ELAPSED=1; [[ "$ELAPSED" -ge 2 ]] && [[ "$ELAPSED" -le 4 ]]; echo OK'
+ ELAPSED=1
+ [[ 1 -ge 2 ]]
+ echo OK
OK
```